utils/helpers.go

```diff
@@ -648,3 +650,58 @@ func containsErrorMessageNameTaken(msg string) bool {
 func Timestamp() string {
 	return time.Now().Format("20060102150405")
 }
+
+func CheckGitignore(path string) (bool, error) {
+	logger.Debug("Checking .gitignore file for existence of Vulcan files")
+	path = path + "/.gitignore"
```

Comment:
I believe this change is necessary because the path separator differs between Windows and Linux.
```suggestion
	path = filepath.Join(path, "/.gitignore")
```

not forget to import package `filepath`